riot/client.go

```diff
@@ -4,6 +4,7 @@ package riot
 
 import (
 	log "github.com/sirupsen/logrus"
+	"github.com/yigithanbalci/golio/riot/val"
```

Comment:
```suggestion
	"github.com/KnutZuidema/golio/riot/val"
```

The package name is wrong in most files.